### DISCUSSION TITLE
Specials char while adding items

### DISCUSSION THREAD

**Discussion by pINKmUNster** [pnp/pnpcore#430] [Q&A]
Hello, 

do someone know how to manage specials chars  while adding items into a listitem?

I'm doing stuff like this : 

```
// Fill a dictionary with fields and their value
Dictionary<string, object> item = new Dictionary<string, object>()
{
    { "Chargé", "Item1" },
    { "Field A", 25 }
};

// Persist the item
var addedItem = await myList.Items.AddAsync(item);
```

And i'm facing  a 400 issue 

```
HttpResponseCode: 400
Code: System.ArgumentException
Message: Column 'Chargé' does not exist. It may have been deleted by another user.  /sites/*/Lists/List
```

Thank in advance

**Comment by jansenbe** [comment]
@pINKmUNster : can you confirm you've a field with internal name `Chargé` in your list?

**Comment by pINKmUNster** [comment]
@jansenbe No, i have Charg_x00e9_  , if I understand well I need to use the internal name instead of the title , to add or update items in a list item?

**Reply by jansenbe** [reply]
Indeed
